doc/how-to/perform-a-release.md

```diff
@@ -0,0 +1,69 @@
+# Perform a release
+
+## Prerequisites
+
+- Be a part of the Ministry of Justice GitHub organisation. This should be done
+  as part of your team onboarding process.
+- Be a part of the `accredited-programmes-team` team. This should be done as
+  part of your team onboarding process.
+
+## Releasing to the development environment
+
+This will be performed automatically on a merge to the `main` branch providing
+the `integration_test`, `build_docker`, `helm_lint` and `unit_test` jobs pass on
+CI.
+
+## Releasing to the preprod environment
+
+Once the `deploy_dev` CI job has passed on CI after merging a branch to `main`,
+the pipeline will hold and await manual approval (as documented in the Circle CI
+documentation
+[here](https://circleci.com/docs/workflows/#holding-a-workflow-for-a-manual-approval))
+at the `deploy_preprod` step.
+
+Perform any necessary manual testing on the development environment and ensure
+your changes are backwards compatible.
```

Comment:
Is it worth linking to the ADR where it explains what you mean by backwards compatible?

Reply:
I've added a bit more description here